coq-vst.opam

```diff
@@ -19,11 +19,7 @@ license: "https://raw.githubusercontent.com/PrincetonUniversity/VST/master/LICEN
 build: [make "BITSIZE=64" "-j%{jobs}%" "msl" "sepcomp" "veric" "floyd"]
 run-test: [make "BITSIZE=64" "-j%{jobs}%" "progs"]
 install: [
-	["mkdir" "-p" "%{lib}%/coq/user-contrib/VST"]
-	["cp" "-r" "msl" "%{lib}%/coq/user-contrib/VST/"]
-	["cp" "-r" "veric" "%{lib}%/coq/user-contrib/VST/"]
-	["cp" "-r" "floyd" "%{lib}%/coq/user-contrib/VST/"]
-	["cp" "-r" "sepcomp" "%{lib}%/coq/user-contrib/VST/"]
+	make "install"
```

Comment:
Won't you need BITSIZE=64 here?

Reply:
Oh yes I should. I was confused by the CompCert directory logic and thought `BITSIZE=64` was the default.
https://github.com/PrincetonUniversity/VST/blob/e4211e15463cc864a06c70059c895b9257ff486a/Makefile#L80-L87